packages/@react-types/form/src/index.d.ts

```diff
@@ -60,7 +60,7 @@ export interface FormProps extends AriaLabelingProps {
    * A nonstandard attribute used by iOS Safari that controls how textual form elements should be automatically capitalized.
    * See [MDN](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/form#autocapitalize).
    */
-  autoCapitalize?: 'off' | 'on',
+  autoCapitalize?: 'off' | 'none' | 'on' | 'sentences' | 'words' | 'characters',
```

Comment:
I changed props type.
but, I'm not confident this is correct and would like to hear opinions on it.

ref: 
- https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input#autocapitalize
- https://html.spec.whatwg.org/multipage/interaction.html#autocapitalization